src/components/Comments/Comment.js

```diff
@@ -46,6 +55,15 @@ const Comment = ({
     level,
     updateVote
   });
+  const [loading, setLoading] = React.useState({
+    loadingDm: false,
+    loadingDmAnon: false
+  });
+
+  const [profile] = React.useContext(Context).profile;
+  const {createSignChat} = useCreateChat();
+  const {sendMessageDM} = useDMMessage();
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New state variables and hooks are introduced here. It's good to see that loading states are being handled separately for different types of operations (`loadingDm` and `loadingDmAnon`). The usage of context and custom hooks can help to simplify the logic and improve readability.
```diff
+ const [loading, setLoading] = React.useState({
+   loadingDm: false,
+   loadingDmAnon: false
+ });
+ 
+ const [profile] = React.useContext(Context).profile;
+ const {createSignChat} = useCreateChat();
+ const {sendMessageDM} = useDMMessage();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->